src/Navigator/Browser.Navigator.fsproj

```diff
@@ -2,8 +2,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <PackageId>Fable.Browser.Navigator</PackageId>
-    <Version>2.0.0</Version>
```

Comment:
For information, 

when working on fable-browser package you should not change the fsproj version.

It is done automatically by the build system and if the version detected in the fsproj is the same as the one in the RELEASE_NOTES then the build system will skip this package.

I believe that the case for almost all the packages under fable-compiler org and across others repository too (at least mine).

Reply:
@MangelMaxime I always forgot about that :(